teams/teams-ps/teams/New-CsBatchPolicyPackageAssignmentOperation.md

```diff
@@ -53,7 +53,7 @@ Accept wildcard characters: False
 
 ### -PackageName
 
-The name of a specific policy package to apply. All policy package names can be found by running Get-CsPolicyPackage.
+The name of a specific policy package to apply. All policy package names can be found by running Get-CsPolicyPackage. To remove the currently assigned package, use $null or an empty string "". This will not remove any policy assignments.
```

Comment:
Small verbiage edit: "This will not remove any policy assignments, just the package assigned value"